packaging/crio.conf.d/microshift_amd64.conf

```diff
@@ -2,8 +2,19 @@
 selinux = true
 conmon = ""
 conmon_cgroup = "pod"
+default_env = [
+    "NSS_SDB_USE_CACHE=no",
+]
 cgroup_manager = "systemd"
 default_runtime = "crun"
+default_sysctls = [
+    "net.ipv4.ping_group_range=0 2147483647",
+]
+manage_ns_lifecycle = true
+absent_mount_sources_to_reject = [
+    "/etc/hostname",
+]
+drop_infra_ctr = true
```

Comment:
this value is the default, why are we specifying manually?

Reply:
I copied it from openshift crio config: https://github.com/openshift/machine-config-operator/blob/cbd7d9514d6a03c84192058edae7a6ff4f9d6941/templates/master/01-master-container-runtime/_base/files/crio.yaml#L34

removed it in the new PR version.